legacy/shared/rpm-utils.sh

```diff
@@ -175,7 +175,8 @@ function rpm_install()
 
 function rpm_extract()
 {
-        __yum_call --downloadonly --downloaddir=$RPM_TMPDIR "${RPM_EXTRACT[@]}"
+        yumdownloader --enablerepo=latest --disablerepo='kernel-cki' \
```

Comment:
I'm not sure if we have the `kernel-cki` name documented/guaranteed anywhere, so I'd rather just disable all other repos. Like this:

    yumdownloader --disablerepo='*' --enablerepo=latest \
                  --downloaddir=$RPM_TMPDIR "${RPM_EXTRACT[@]}"

Reply:
@spbnick Good catch, resolved.